src/matvis/coordinates.py

```diff
@@ -13,6 +13,18 @@
 from pyuvdata.uvbeam import UVBeam
 from typing import Literal
 
+from . import HAVE_GPU
+
+if HAVE_GPU:
+    import cupy as cp
+
+    get_array_module = cp.get_array_module
+else:
+
+    def get_array_module(*x):
```

Comment:
Does it actually work to define a function inside the if-else?

Reply:
absolutely!

Reply:
Nice!